packages/web-app-files/src/components/Shares/SharedWithMeSection.vue

```diff
@@ -168,15 +160,17 @@ export default defineComponent({
     displayThumbnails: {
       type: Boolean,
       default: true
+    },
+    sideBarOpen: {
+      type: Boolean,
+      default: false
+    },
+    fileListHeaderY: {
+      type: Number,
+      default: 0
     }
   },
   setup() {
-    const { fileListHeaderY, selectedResourcesIds, selectedResources } = useResourcesViewDefaults<
```

Comment:
decided to remove the `useResourcesViewDefaults` composable here, because that one now subscribes to some event bus topics - it should only be used in a view (once per view). Additionally I moved the variables and methods related to `selected resources` into its own composable (which is then spreaded into the useResourcesViewDefaults composable) so that we can continue to use that here. Remaining composable refs `sideBarOpen` (new) and `fileListHeaderY` (existing) have been moved to props and need to be passed down by the shared with me view.